### DISCUSSION TITLE
Frame WalletConnect Project

### DISCUSSION THREAD

**Discussion by goosewobbler** [WalletConnect/walletconnect-monorepo#2356] [Developer Support] [answered]
Hi,

[Frame](https://frame.sh) dev here, we would like to integrate WalletConnect but we still do not have control of our project.  How do we move the request along?

The project ID for Frame is: a9751f17a3292f2d1493927f0555603d69e9a3fcbcdf5626f01b49afa21ced91

Thanks

**Comment by finessevanes** [comment]
Looks like your project has been approved!

**Reply by goosewobbler** [reply]
Approval isn't the same as ownership, it was set up by someone else and we don't own it.

**Reply by finessevanes** [reply, marked as answer]
Gotcha - please email [cloud@walletconnect.com](mailto:cloud@walletconnect.com) to verify ownership. Please be sure to use the right email domain and include information from your project.

**Reply by goosewobbler** [reply]
Thanks, will give it a go.  I think we already emailed support@walletconnect.com as per the explorer [docs](https://docs.walletconnect.com/2.0/cloud/explorer#submitting-a-project).
